.golangci.yaml

```diff
@@ -0,0 +1,6 @@
+run:
+  build-tags:
+    # Keep these tags in sync with CONTAINERS_STORAGE_THIN_TAGS in build.sh
+    - containers_image_openpgp
+    - exclude_graphdriver_btrfs
+    - exclude_graphdriver_devicemapper
```

Comment:
This is a configuration file for `golangci-lint` in this CI test.

https://github.com/osbuild/bootc-image-builder/blob/f5e218ae5e3a985c6a0891dc3959a2056136db59/.github/workflows/tests.yml#L41-L46